maxima/g0/neutrals/sigma-cx-calc.mac

```diff
@@ -0,0 +1,141 @@
+/* Generate CX cross section (vSigmaCX) kernels for charge
+   exchange operator */
+
+load("modal-basis");
+load("out-scripts");
+load(stringproc)$
+fpprec : 24$
+
+/* Serendipity basis. */
+maxPolyOrder_Ser : 2$
+
+declareWriteCExprsC(lhs, rhs) := block([expr],
+  expr : float(expand(rhs)),
+  if expr # 0.0 then (
+    ep : string(expr),
+    for i : 0 thru 60 do (
+    	ep : ssubst(string(pow(u_ion[i],2.)),string(u_ion[i]^2),ep),
+    	ep : ssubst(string(pow(u_neut[i],2.)),string(u_neut[i]^2),ep)	
+    ),
+    printf(fh, "  double ~a = ~a; ~%", lhs, ep)
+  )
+)$
+
+writeCExprsCs(lhs, rhs) := block([expr],
+  expr : float(expand(rhs)),
+  for i : 1 thru length(expr) do (	
+    if expr[i] # 0.0 then (
+      ep : string(expr[i]),
+      ep : ssubst("log(","ln(",ep),	
+      printf(fh, "  ~a = ~a; ~%", lhs[i-1], ep)	
+    )
+  )
+)$
+
+writeCExprsC(lhs, rhs) := block([expr],
+  expr : float(expand(rhs)),
+  if expr # 0.0 then (
+     ep : string(expr),
+     printf(fh, "  ~a = ~a; ~%", lhs, ep)	
+  )
+)$
+
+/* vSigmaCX for charge exchange cross section */
+calcSigmaCXcellAv(fh, funcNm, cdim, vdim, basisFun, pMax) := block([],
+  kill(varsC, varsP, basisC, basisP),
+  modNm : sconcat("basis-precalc/basis", basisFun, cdim, "x", vdim, "v"),
```

Comment:
@tnbernard please don't load basis loading files yourself. Use the function to load basis in modal-basis.mac

Reply:
Fixed in latest commit.